### PR TITLE
feat(add-profile): EddieHubCommunity

### PR DESCRIPTION
**Pull request by krishguptadev**
RR: @eddiejaoude 

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/703"></a>

### CODE REVIEW

**Review by nhcarrigan — APPROVED** [review]
Happy to merge as-is. We should circle back and add milestones at some point, like the date the org was founded, when we reached 1000 members, etc.

**Comment by krishguptadev** [comment]
> Happy to merge as-is. We should circle back and add milestones at some point, like the date the org was founded, when we reached 1000 members, etc.

I recently joined the server so don't know about this stuff, but I'll ask Eddie...

- When did our comm win the Nova Price?
- When did this comm started?

and such...

**Comment by eddiejaoude** [comment]
I think the details are

- EddieHub started on November 2020
- Reach 1000 members on Discord March 2021

**Comment by krishguptadev** [comment]
> I think the details are
> 
> * EddieHub started on November 2020
> * Reach 1000 members on Discord March 2021

and nova?

**Review by Panquesito7 — APPROVED** [review]
Amazing! 🚀

**Review by Vyvy-vi — APPROVED** [review]
looks good to me!

File: public/data/EddieHubCommunity.json [hunk 1]
```diff
@@ -0,0 +1,38 @@
+{
+  "name": "EddieHubCommunity",
+  "bio": "Community of inclusive Open Source people - Collaboration 1st, Code 2nd!",
+  "avatar": "https://github.com/EddieHubCommunity.png",
+  "links": [
+    {
+      "name": "Visit Website",
+      "url": "https://eddiehub.org",
```
Comment (Panquesito7) [inline public/data/EddieHubCommunity.json:8]: The current link doesn't work 👀 no idea why.
```suggestion
      "url": "https://www.eddiehub.org/",
```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

**Review by ssensalo — APPROVED** [review]

File: public/data/EddieHubCommunity.json (hunk 1 above)
Comment (krishguptadev) [reply public/data/EddieHubCommunity.json:8]: I think www. Redirects to the without www. Version

Comment (krishguptadev) [reply public/data/EddieHubCommunity.json:8]: I think it is the https

```suggestion
      "url": "http://eddiehub.org",
```

**Review by Panquesito7 — APPROVED** [review]
LGTM 🚀 Thank you for your contribution! 😄👍

**Pull request merged**